okhttp/src/main/java/okhttp3/OkHttpClient.java

```diff
@@ -181,9 +179,8 @@ public void apply(ConnectionSpec tlsConfiguration, SSLSocket sslSocket, boolean
         tlsConfiguration.apply(sslSocket, isFallback);
       }
 
-      @Override public HttpUrl getHttpUrlChecked(String url)
-          throws MalformedURLException, UnknownHostException {
-        return HttpUrl.getChecked(url);
+      @Override public HttpUrl getHttpUrlChecked(String url) {
```

Comment:
We can eliminate this override and the method in the interface and call directly into the public method now.